client/package.json

```diff
@@ -30,7 +30,7 @@
 		"styled-components": "^5.1.1"
 	},
 	"scripts": {
-		"start": "react-scripts --openssl-legacy-provider start",
+		"start": "react-scripts start",
```

Comment:
This reverts a fix made in 0a865946f2b503bc07e7f415995d6597b743ade5... did the fix break your version?

Reply:
Undoing this now